Doc/library/sqlite3.rst

```diff
@@ -221,6 +223,13 @@ creating a new cursor, then querying the database:
    >>> print(f'The highest scoring Monty Python movie is {title!r}, released in {year}')
    The highest scoring Monty Python movie is 'Monty Python and the Holy Grail', released in 1975
 
+.. testcleanup::
+
+   new_con.close()
+
+   import os
+   os.remove("tutorial.db")
```

Comment:
Would it be possible to use support.os_helper.unlink() here?

Reply:
I don't know. There is no precedent for it elsewhere in `Doc/`.

Reply:
Resolved with #117604 for now.